frappe/core/doctype/docfield/docfield.json

```diff
@@ -526,13 +527,20 @@
   {
    "fieldname": "column_break_35",
    "fieldtype": "Column Break"
+  },
+  {
+   "default": "0",
+   "depends_on": "eval:doc.fieldtype===\"Tab Break\"",
+   "fieldname": "show_dashboard",
+   "fieldtype": "Check",
+   "label": "Show Dashboard"
```

Comment:
we need to add this in Custom Field as well. 

Also, I understand this is a UX level fix but is it worth adding an additional column just for this which will be used very rarely? IMO using "options" for this configuration is not that bad.

Reply:
Options are terrible UX and not discoverable, also it was undocumented. A boolean field is essentially cheap.

Reply:
Added in customize field 👍